src/main/java/org/skriptlang/skript/lang/arithmetic/Arithmetics.java

```diff
@@ -129,6 +129,23 @@ public static <L, R, T> OperationInfo<L, R, T> lookupOperationInfo(Operator oper
 		return info != null ? info.getConverted(leftClass, rightClass, returnType) : null;
 	}
 
+	public static <L, R> @Nullable OperationInfo<L, R, ?> lookupOperationInfo(
+		Operator operator,
+		Class<L> leftClass,
+		Class<R> rightClass,
+		Class<?> ... possibleReturnTypes
+	) {
+		OperationInfo<L, R, ?> info = lookupOperationInfo(operator, leftClass, rightClass);
+		if (info == null)
+			return null;
+		for (Class<?> returnType : possibleReturnTypes) {
```

Comment:
You probably want to first check that the return type of the operation info is not one of `possibleReturnTypes`. That is, we want to avoid converting if possible.